Gemfile

```diff
@@ -23,6 +23,7 @@ gem 'jbuilder', '~> 2.0'
 # bundle exec rake doc:rails generates the API under doc/api.
 gem 'sdoc', '~> 0.4.0', group: :doc
 
+gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
```

Comment:
never used this before but looks useful 👍 

based on what I read up on you need the `tzinfo` gem for `tzinfo-data` to be useful? should we be adding `tzinfo`? not completely sure though